lesson-7/7lesson_dz.py

```diff
@@ -0,0 +1,26 @@
+phone_list = {
```

Comment:
It is better to put all this code inside of `while True:` loop to have your program running

Reply:
Done